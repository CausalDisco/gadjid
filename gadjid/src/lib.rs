// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
