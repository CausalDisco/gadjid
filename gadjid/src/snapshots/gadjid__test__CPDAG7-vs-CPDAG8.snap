---
source: src/lib.rs
expression: "test(\"10007.CPDAG-100\", \"10008.CPDAG-100\")"
---
g_guess: 10008.CPDAG-100
g_true: 10007.CPDAG-100
ts:
  - 10
  - 17
  - 36
  - 20
  - 96
ys:
  - 38
  - 70
  - 14
  - 64
  - 69
zs:
  - - 1
    - 2
    - 4
    - 6
    - 7
    - 8
    - 9
  - - 0
    - 2
    - 5
    - 7
    - 10
    - 11
    - 13
    - 14
  - - 2
    - 3
    - 10
    - 12
    - 13
    - 14
    - 15
    - 17
    - 18
    - 20
    - 23
    - 32
    - 33
    - 35
  - - 0
    - 3
    - 9
    - 13
  - - 1
    - 3
    - 4
    - 5
    - 6
    - 7
    - 9
    - 10
    - 11
    - 13
    - 14
    - 16
    - 17
    - 18
    - 19
    - 20
    - 24
    - 27
    - 30
    - 31
    - 32
    - 35
    - 36
    - 37
    - 39
    - 41
    - 42
    - 43
    - 44
    - 45
    - 46
    - 48
    - 50
    - 52
    - 53
    - 54
    - 56
    - 57
    - 58
    - 59
    - 66
    - 67
    - 68
    - 73
    - 75
    - 77
    - 79
    - 84
    - 86
    - 89
    - 90
    - 91
    - 92
    - 95
nams:
  - []
  - []
  - []
  - []
  - []
nvas:
  - - 0
    - 1
    - 2
    - 3
    - 4
    - 5
    - 6
    - 7
    - 8
    - 9
    - 11
    - 14
    - 15
    - 16
    - 17
    - 18
    - 19
    - 20
    - 21
    - 22
    - 23
    - 24
    - 25
    - 26
    - 27
    - 28
    - 29
    - 30
    - 31
    - 32
    - 33
    - 34
    - 35
    - 36
    - 37
    - 38
    - 39
    - 40
    - 41
    - 42
    - 43
    - 44
    - 45
    - 46
    - 47
    - 48
    - 49
    - 50
    - 51
    - 52
    - 53
    - 54
    - 55
    - 56
    - 57
    - 58
    - 59
    - 60
    - 61
    - 62
    - 63
    - 64
    - 65
    - 66
    - 67
    - 68
    - 69
    - 70
    - 71
    - 72
    - 73
    - 74
    - 75
    - 76
    - 77
    - 78
    - 79
    - 80
    - 81
    - 82
    - 83
    - 84
    - 85
    - 86
    - 87
    - 88
    - 89
    - 90
    - 91
    - 92
    - 93
    - 94
    - 95
    - 96
    - 97
    - 98
    - 99
  - - 0
    - 2
    - 5
    - 7
    - 10
    - 11
    - 13
    - 14
  - - 2
    - 3
    - 10
    - 12
    - 13
    - 14
    - 15
    - 17
    - 18
    - 20
    - 23
    - 32
    - 33
    - 35
  - - 0
    - 3
    - 9
    - 13
  - - 1
    - 3
    - 4
    - 5
    - 6
    - 7
    - 9
    - 10
    - 11
    - 13
    - 14
    - 16
    - 17
    - 18
    - 19
    - 20
    - 24
    - 27
    - 30
    - 31
    - 32
    - 35
    - 36
    - 37
    - 39
    - 41
    - 42
    - 43
    - 44
    - 45
    - 46
    - 48
    - 50
    - 52
    - 53
    - 54
    - 56
    - 57
    - 58
    - 59
    - 66
    - 67
    - 68
    - 73
    - 75
    - 77
    - 79
    - 84
    - 86
    - 89
    - 90
    - 91
    - 92
    - 95
pa_true_1st_T:
  - 0
  - 1
  - 2
  - 4
  - 6
  - 7
  - 8
  - 9
an_true_1st_T:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
  - 10
ch_true_1st_T:
  - 11
  - 13
  - 14
  - 15
  - 16
  - 17
  - 18
  - 19
  - 21
  - 22
  - 23
  - 24
  - 25
  - 28
  - 29
  - 32
  - 33
  - 35
  - 36
  - 37
  - 41
  - 42
  - 45
  - 47
  - 49
  - 50
  - 51
  - 53
  - 54
  - 56
  - 57
  - 59
  - 61
  - 63
  - 64
  - 65
  - 68
  - 70
  - 77
  - 81
  - 83
  - 88
  - 89
  - 91
  - 93
  - 94
  - 96
  - 99
de_true_1st_T:
  - 10
  - 11
  - 13
  - 14
  - 15
  - 16
  - 17
  - 18
  - 19
  - 20
  - 21
  - 22
  - 23
  - 24
  - 25
  - 26
  - 27
  - 28
  - 29
  - 30
  - 31
  - 32
  - 33
  - 34
  - 35
  - 36
  - 37
  - 38
  - 39
  - 40
  - 41
  - 42
  - 43
  - 44
  - 45
  - 46
  - 47
  - 48
  - 49
  - 50
  - 51
  - 52
  - 53
  - 54
  - 55
  - 56
  - 57
  - 58
  - 59
  - 60
  - 61
  - 62
  - 63
  - 64
  - 65
  - 66
  - 67
  - 68
  - 69
  - 70
  - 71
  - 72
  - 73
  - 74
  - 75
  - 76
  - 77
  - 78
  - 79
  - 80
  - 81
  - 82
  - 83
  - 84
  - 85
  - 86
  - 87
  - 88
  - 89
  - 90
  - 91
  - 92
  - 93
  - 94
  - 95
  - 96
  - 97
  - 98
  - 99
poss_de_true_1st_T:
  - 10
  - 11
  - 13
  - 14
  - 15
  - 16
  - 17
  - 18
  - 19
  - 20
  - 21
  - 22
  - 23
  - 24
  - 25
  - 26
  - 27
  - 28
  - 29
  - 30
  - 31
  - 32
  - 33
  - 34
  - 35
  - 36
  - 37
  - 38
  - 39
  - 40
  - 41
  - 42
  - 43
  - 44
  - 45
  - 46
  - 47
  - 48
  - 49
  - 50
  - 51
  - 52
  - 53
  - 54
  - 55
  - 56
  - 57
  - 58
  - 59
  - 60
  - 61
  - 62
  - 63
  - 64
  - 65
  - 66
  - 67
  - 68
  - 69
  - 70
  - 71
  - 72
  - 73
  - 74
  - 75
  - 76
  - 77
  - 78
  - 79
  - 80
  - 81
  - 82
  - 83
  - 84
  - 85
  - 86
  - 87
  - 88
  - 89
  - 90
  - 91
  - 92
  - 93
  - 94
  - 95
  - 96
  - 97
  - 98
  - 99
prop_an_true_1st_T_and_1st_Y:
  - 0
  - 1
  - 2
  - 3
  - 4
  - 5
  - 6
  - 7
  - 8
  - 9
  - 11
  - 12
  - 13
  - 14
  - 15
  - 16
  - 17
  - 18
  - 19
  - 20
  - 21
  - 22
  - 23
  - 24
  - 25
  - 26
  - 27
  - 28
  - 29
  - 30
  - 31
  - 32
  - 33
  - 34
  - 35
  - 36
  - 37
  - 38
shd:
  - 0.00808080808080808
  - 40
oset_aid:
  - 0.008686868686868687
  - 86
parent_aid:
  - 0.14505050505050504
  - 1436
ancestor_aid:
  - 0
  - 0
